.circleci/config.yml

```diff
@@ -1308,6 +1308,25 @@ executors:
     working_directory: /repo
 
 workflows:
+  GCP-Personal-Dev:
+    jobs:
+      - build-platform-and-services:
+          filters:
+            branches:
+              only:
+                - developers-branch-to-run-tests
+      - jrs-regression:
+          context: Slack
+          result_path: results/Personal
+          config_type: "personal"
+          workflow-name: "GCP-Personal-DevName"
```

Comment:
This lines may also need to changes when merging to master,

Reply:
As mentioned above, this will not be run on master. It is workflow for helping devs to run their personal JSON when needed

Reply:
Removed the workflow @ljianghedera